src/features/common/LandingVideo/styles.module.scss

```diff
@@ -129,4 +133,7 @@
     right: 125px;
     flex-direction: row;
   }
+  .embed_playButton{
+    top:50px;
+  }
```

Comment:
The play button is a little lower than the project list on the left. I think this should be

.embed_playButton{
    top:20px;
    right: 125px;
}

as there is a margin of 20px above the main container/embedContainer